src/containers/List/List.tsx

```diff
@@ -259,7 +259,7 @@ export const List = ({
 
   useEffect(() => {
     refetchCount();
-  }, [filterPayload, searchVal, filters]);
```

Comment:
Same as above

Reply:
refetchcount was getting called even if the filterPayload and filter were not changing while closing or opening the side drawer. Removing these doesnt seem to cause any issue with the existing features.